assets/scss/abstracts/_index.scss

```
@@ -0,0 +1,127 @@
+
+$paper: #fafafa;
+$paper-2: #ffffff;
+$line: #e5e7eb;
+$line-2: #d1d5db;
+
+// Accent system
+$accent: $amber-600;
+$accent-2: $amber-500;
+
+// Typography
+$font-display: "Newsreader", "Playfair Display", Georgia, "Times New Roman", serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)